apps/new/widget/page/projects/Editor.jsx

```diff
@@ -341,6 +359,15 @@ const Container = styled.div`
     }
   }
 
+  .destructive {
```

Comment:
should destructive be a variant? Like "primary", but instead is "danger"?

Reply:
yeah good suggestion, I checked the btn component and couldn't find one, I was going to create a task for it.